CPackLists.txt

```diff
@@ -30,6 +30,8 @@
             DESTINATION ${CMAKE_INSTALL_HEADERDIR}/g3log
             COMPONENT headers)
 
+   INSTALL( FILES ${PROJECT_SOURCE_DIR}/cmake/G3LOGConfig.cmake
+            DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/G3LOG)
```

Comment:
I confirm that it works in both Mac OS X and Linux. CMake in Mac OSX comes from macport. 
But it really doesn't matter I believe. It is CMake version matters.

See doc : https://cmake.org/cmake/help/v3.0/manual/cmake-packages.7.html#package-configuration-file

There is a sample below how to create/install package.